pontoon/base/signals.py

```diff
@@ -17,3 +22,78 @@ def project_locale_added(sender, **kwargs):
         project = project_locale.project
         project.has_changed = True
         project.save(update_fields=['has_changed'])
+
+
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    Managers should be able to translate and manage locale.
+    """
+    instance = kwargs['instance']
+
+    if kwargs['raw'] or instance.translators_group:
```

Comment:
This is a nit, but checking for `is None` instead of relying on the fact that `None` is falsy is a good general practice in case you ever run into an object that is falsy for weird reasons.